db/schema.rb

```diff
@@ -188,6 +188,8 @@
     t.string "utm_source"
     t.string "utm_medium"
     t.string "utm_term"
+    t.string "coa_by_authenticating_body"
+    t.string "coa_by_gallery"
```

Comment:
To match with the [gravity definition](https://github.com/artsy/gravity/blob/50aeb677c8cddc401161b65bffa09dba57167f82/app/models/util/artwork_condition.rb), these should actually be booleans both here and in the GraphQL type :)

Reply:
Oh thanks! That's even easier. Should there be a default value like `false` if none is given? Or is `nil` okay..?

Reply:
I think `nil` is better because it more clearly indicates that we don't know whether or not a work has a certain type of COA - `false` implies that we're certain it _doesn't_ have that type of COA.